src/index.html

```diff
@@ -11,7 +11,200 @@
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+
+          <input
+            type="date"
+            class="field"
+            name="birthDate"
+            required
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+
+          <input
+            type="checkbox"
+            class="field"
+            name="term"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+
+          <input
+            type="email"
+            class="field"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+
+          <input
+            type="password"
+            class="field"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              class="field"
+              name="loveCats"
+              value="yes"
+              required
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              class="field"
+              name="loveCats"
+              value="no"
+              required
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+
+          <input
+            type="color"
+            class="field"
+            name="color"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            class="field"
+            name="time"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+
+          <select name="car" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+
+          <input
+            type="range"
+            class="field"
+            name="rate"
+            required
+            min="0"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+
+          <textarea name="comments" cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea name="comments" cols="20"
            rows="2"
          ></textarea>
```
![image](https://user-images.githubusercontent.com/97547824/200045733-c55d44c7-c5bb-4a06-a198-71a22cde9ce4.png)